app/scripts/main.js

```diff
@@ -6,18 +6,19 @@ $(document).ready(function() {
   // ==============
 
   // Replace with your own values
-  var APPLICATION_ID = 'latency';
-  var SEARCH_ONLY_API_KEY = '6be0576ff61c053d5f9a3225e2a90f76';
-  var INDEX_NAME = 'bestbuy';
-  var HITS_PER_PAGE = 10;
+  var APPLICATION_ID       = 'latency';
```

Comment:
If you want to switch the style to indent spaces, do it in another PR, it makes the diff unreadable.

I do not recommend indenting spaces for that same reason (unreadable diffs when adding a new longer variable).

We stopped doing it in the JS projects.

Can you remove it?